src/qt/rpcconsole.cpp

```diff
@@ -776,20 +776,15 @@ void RPCConsole::setFontSize(int newSize)
 
     // clear console (reset icon sizes, default stylesheet) and re-add the content
     float oldPosFactor = 1.0 / ui->messagesWidget->verticalScrollBar()->maximum() * ui->messagesWidget->verticalScrollBar()->value();
-    clear(false);
+    clear(/* keep_prompt */ true);
     ui->messagesWidget->setHtml(str);
     ui->messagesWidget->verticalScrollBar()->setValue(oldPosFactor * ui->messagesWidget->verticalScrollBar()->maximum());
 }
 
-void RPCConsole::clear(bool clearHistory)
```

Comment:
In the future, please keep the existing param so calls to eg `clear(true)` don't silently change behaviour...

Reply:
I don't follow... In d2cc3390054616c73f72a59f864700f6de14067b `RPCConsole::clear()` has no parameters.
Why keep the existing useless param?

Reply:
In case another PR were to use the bool param, this change would silently result in different behaviour when the two are merged together. Using an enum class would also solve it.

Reply:
> Using an enum class would also solve it.

Agree. https://github.com/bitcoin/bitcoin/issues/21684